runtime/mount/kernel.py

```diff
@@ -448,7 +449,9 @@ def serialize_plotly_figure(x: BaseFigure):
 
     # note(maximsmol): plotly itself does a bunch of escaping to avoid XSS
     # when embedding directly into HTML. we never do that so we don't care
-    return pio_json.clean_to_json_compatible(res, modules=modules)
+    return pio_json.clean_to_json_compatible(
```

Comment:
plotly allows datetime for `orjson`: https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/plotly/io/_json.py#L165